src/mme-app/mmeThreads.h

```diff
@@ -23,7 +23,7 @@
 #include <interfaces/mmeIpcInterface.h>
 #include <mme_app.h>
 
-#define DATA_BUF_SIZE 1024
+#define DATA_BUF_SIZE 4096
```

Comment:
This is what I see discrepancy. DATA_BUF_SIZE is defined 1024 in other file. I have already commented that place.

Reply:
This is the IPC message buffer size..concurrent to BUFFER_LEN in s1ap/main.c